JS-Algorithms/challenges.js

```diff
@@ -71,6 +106,13 @@ The last 3 digits for the sum of powers from 1 to 10 is "317"
 
 const ownPower = (number, lastDigits) => {
 	// YOUR CODE HERE...
+	let sum = 0;
+	for (let i = 1; i <= number; i++) {
+		sum += i ** i;
+	}
```

Comment:
I feel this can be a perfect use-case for [`Array.prototype.reduce`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce)